runtime/nls/j9cl/j9jcl.nls

```diff
@@ -580,3 +580,11 @@ J9NLS_JCL_CRIU_LOADING_OPTIONS_FILE_FAILED.explanation=CRIUSupport::checkpointJV
 J9NLS_JCL_CRIU_LOADING_OPTIONS_FILE_FAILED.system_action=The JVM will throw a JVMRestoreException.
 J9NLS_JCL_CRIU_LOADING_OPTIONS_FILE_FAILED.user_response=Check the arguments to CRIUSupport::registerRestoreOptionsFile.
 # END NON-TRANSLATABLE
+
+J9NLS_JCL_CRIU_RESTORE_INITIALIZE_TRACE_FAILED=The JVM could not initialize trace with the restore option file
```

Comment:
`The JVM could not initialize trace upon restoring the JVM`

Options can be specified by env var or optinos file

Reply:
Updated